packages/client/defaultDeps.json

```diff
@@ -172,8 +172,6 @@
     "@mui/material/Typography",
     "@mui/material/colors",
     "@mui/material/styles",
```

Comment:
i had no idea we still had defaultDeps. we can remove this file entirely i'm pretty sure